test/Microsoft.AspNetCore.Mvc.RazorPages.Test/Internal/PageActionInvokerProviderTest.cs

```diff
@@ -299,6 +362,238 @@ public void OnProvidersExecuting_UpdatesEntriesWhenActionDescriptorProviderColle
             Assert.NotSame(entry1, entry2);
         }
 
+        [Fact]
+        public void OnProviderExecuting_PagePopulates()
+        {
+            // Arrange
+            var type = typeof(TestSetPageWithModel);
+
+            var descriptor = new PageActionDescriptor
+            {
+                RelativePath = "Path1",
+                FilterDescriptors = new FilterDescriptor[0],
+            };
+            var descriptorCollection1 = new ActionDescriptorCollection(new[] { descriptor }, version: 1);
+            var descriptorCollection2 = new ActionDescriptorCollection(new[] { descriptor }, version: 2);
+            var actionDescriptorProvider = new Mock<IActionDescriptorCollectionProvider>();
+            actionDescriptorProvider.SetupSequence(p => p.ActionDescriptors)
+                .Returns(descriptorCollection1)
+                .Returns(descriptorCollection2);
+
+            var loader = new Mock<IPageLoader>();
+            loader.Setup(l => l.Load(It.IsAny<PageActionDescriptor>()))
+                .Returns(CreateCompiledPageActionDescriptor(descriptor, type));
+            var invokerProvider = CreateInvokerProvider(
+                 loader.Object,
+                 actionDescriptorProvider.Object);
+            var context = new ActionInvokerProviderContext(
+                new ActionContext(new DefaultHttpContext(), new RouteData(), descriptor));
+
+            // Act
+            invokerProvider.OnProvidersExecuting(context);
+
+            // Assert
+            var entry = Assert.Single(invokerProvider.TestCache.Entries);
+
+            Assert.Equal(2, entry.Value.ActionDescriptor.HandlerMethods.Count);
+        }
+
+        [Fact]
+        public void OnProviderExecuting_ModelPopulates()
```

Comment:
I think this test is covered by https://github.com/aspnet/Mvc/blob/d7bb910114e9232494f6a6751646d86ec3996dd7/test/Microsoft.AspNetCore.Mvc.RazorPages.Test/Internal/PageActionInvokerProviderTest.cs#L187-L188